conferenceOwen/src/navigations/TabNavigation.js

```diff
@@ -0,0 +1,67 @@
+import * as React from 'react';
+import {createMaterialTopTabNavigator} from '@react-navigation/material-top-tabs';
+import Upcoming from '../screens/menuUser/UpcomingScreen';
+import Completed from '../screens/menuUser/CompletedScreen';
+import AddConference from '../screens/menuUser/AddScreen';
+import {Colors} from '../themes/Colors';
+import Icons from 'react-native-vector-icons/AntDesign';
+
+const Tab = createMaterialTopTabNavigator();
+
+export default function Navegation() {
+  return (
+    <Tab.Navigator
+      tabBarOptions={{
+        showIcon: true, //Propertie to show icons in the top tab bar
+        activeTintColor: Colors.activeTab, //Color of the selected button
+        inactiveTintColor: Colors.inactiveTab, // Color of the unselected button
+        pressColor: Colors.pressColor,
+        pressOpacity: Colors.pressColor,
+        // Style object for the tab bar
+        style: {
+          height: 80,
+        },
+        //This is the style of the thin line
```

Comment:
Let's try to do not include unnecessary comments like those one, line-by-line commentary makes the code look almost redundant. Those props can be explained by themselves.